src/VisualStudio/Core/Def/Implementation/MoveStaticMembers/StaticMemberSelection.xaml

```diff
@@ -0,0 +1,94 @@
+﻿<UserControl x:Class="Microsoft.VisualStudio.LanguageServices.Implementation.MoveStaticMembers.StaticMemberSelection"
```

Comment:
We should investigate the effort to unify the member selector controls and selectively hide unnecessary components.